vembrane/__init__.py

```diff
@@ -110,6 +112,14 @@ def main():
         choices=["vcf", "bcf", "uncompressed-bcf"],
         help="Output format.",
     )
+    parser.add_argument(
+        "--ann_key",
```

Comment:
Should be `--ann-key` (or even `--annotation-key` since it's a long option anyway), i.e. hyphen instead of underscore.